samples/excel/85-preview-apis/worksheet-page-layout.yaml

```diff
@@ -0,0 +1,198 @@
+order: 5
+id: excel-worksheet-page-layout
+name: Page layout
```

Comment:
Seems like "Page layout" is only part of the story for this snippet...some of the functions control page layout, others control print settings. To improve discoverability for this snippet, would it be too wordy to name it "Page layout and print settings" ?   Or maybe just "Print settings" is sufficient?